Automattic/wp-calypso
# Hide woo branding on partner-signup flow

#### Proposed Changes

* Fixes 13520-gh-Automattic/woocommerce.com
* Replaces the WooCommerce branded oauth login screen when coming from the WordPress.com branded https://woocommerce.com/partner-signup/. 
* Other WooCommerce oauth logins should remain on the Woo branded screen.

#### Testing Instructions

* Visit https://woocommerce.com/partner-signup while logged out
* When redirected to wordpress.com, replace `https://wordpress.com` with `http://calypso.localhost:3000`
* The wordpress.com login screen should be shown instead of the woocommerce oauth screen.
* Repeat for https://woocommerce.com/vendor-signup, where the woo branded login screen should remain.

Alternatively, view the difference between these screens:
* http://calypso.localhost:3000/log-in
* Redirect from /partner-signup/ http://calypso.localhost:3000/log-in?client_id=50916&redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fresponse_type%3Dcode%26client_id%3D50916%26state%3Dsnip%26redirect_uri%3Dhttps%253A%252F%252Fwoocommerce.com%252Fwc-api%252Fwpcom-signin%253Fnext%253Dpartner-signup%26blog_id%3D0%26wpcom_connect%3D1%26wccom-from%26calypso_env%3Dproduction%26redirect_to%3Dhttps%3A%2F%2Fwoocommerce.com%2Fpartner-signup%2F
* Redirect from /vendor-signup/ http://calypso.localhost:3000/log-in?client_id=50916&redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fresponse_type%3Dcode%26client_id%3D50916%26state%3D%26redirect_uri%3Dhttps%253A%252F%252Fwoocommerce.com%252Fwc-api%252Fwpcom-signin%253Fnext%253Dvendor-signup%26blog_id%3D0%26wpcom_connect%3D1%26wccom-from%26calypso_env%3Dproduction%26redirect_to%3Dhttps%3A%2F%2Fwoocommerce.com%2Fvendor-signup%2F
* Reminder, if you get an access denied error at the end of the login flow its probably because you're using one of the above urls with a snipped access key, start the flow from logged out on both wpcom and woocom and visiting woo.com/partner-signup directly.
* Check link back to partners is localized along with login content http://calypso.localhost:3000/log-in/ja

Before
![Screenshot 2022-06-07 at 16-50-54 Log In — WordPress com](https://user-images.githubusercontent.com/811776/172314675-225f82f5-f22d-4214-b6a4-4af0f99822c2.png)

After
![Screenshot 2022-06-07 at 16-50-20 Log In — WordPress com](https://user-images.githubusercontent.com/811776/172314591-abecf577-a337-4dd1-912e-c41e5c625537.png)

Fixes 13520-gh-Automattic/woocommerce.com
